app.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express')
+
+const app = express()
+
+app.use(express.static('public'))
+
+app.get('/', (req, res, next) => {
+  res.sendFile(__dirname + '/views/index.html')
+  })
+
```

Comment:
Esta forma está bien, pero mejor que utilicéis esta otra, ya que con render puedes pasar parametros :

app.get('/', (req, res, next) => res.render('index'))